webservices/legal_docs/index_management.py

```diff
@@ -330,10 +331,10 @@
 }
 
 
-def initialize_legal_docs():
+def initialize_current_legal_docs():
     """
     Initialize Elasticsearch for storing legal documents.
-    Create the `docs` index, and set up the aliases `docs_index` and `docs_search`
+    Create the `docs` index, and set up the aliases `DOCS_INDEX` and `DOCS_SEARCH`
```

Comment:
The aliases are actually called `docs_index` and `docs_search` (lowercase). `DOCS_INDEX` and `DOCS_SEARCH` are merely Python variables that _contain_ the alias names. As the use of these variables doesn't seem to be very helpful (and adds one more level of indirection), we could get rid of them.

Reply:
@vrajmohan ah yes I see that now. I'm not opposed to keeping it the way it is - as it is, would it make future changes easier, or typos easier to spot?

Reply:
With variable -> alias -> index, I think it is confusing. I do think we would benefit from getting rid of it.